src/content_working_group/lib.rs

```diff
@@ -0,0 +1,2422 @@
+        ///
+        /// Notice that working group participants cannot do this.
+        /// Notice that censored or unpublished channel may still be transferred.
+        /// Notice that transfers are unilateral, so new owner cannot block. This may be problematic: https://github.com/Joystream/substrate-runtime-joystream/issues/95
+        pub fn transfer_channel_ownership(origin, channel_id: ChannelId<T>, new_owner: T::MemberId, new_role_account: T::AccountId) {
+
+            // Ensure channel owner has signed
+            let channel = Self::ensure_channel_owner_signed(origin, &channel_id)?;
+
+            // Ensure prospective new owner can actually become a channel owner
+            let (new_owner_as_channel_owner, _next_channel_id) = Self::ensure_can_register_channel_owner_role_on_member(&new_owner, Some(channel_id))?;
+
+            //
+            // == MUTATION SAFE ==
+            //
+
+            // Construct new channel with altered properties
+            let new_channel = Channel {
+                owner: new_owner,
+                role_account: new_role_account.clone(),
+                ..channel
+            };
+
+            // Overwrite entry in ChannelById
+            ChannelById::<T>::insert(channel_id, new_channel);
+
+            // Remove
+            let unregistered_role = <members::Module<T>>::unregister_role(role_types::ActorInRole::new(role_types::Role::ChannelOwner, channel_id)).is_ok();
+
+            assert!(unregistered_role);
+
+            // Dial out to membership module and inform about new role as channe owner.
+            let registered_role = <members::Module<T>>::register_role_on_member(
+                new_owner,
+                &new_owner_as_channel_owner)
+                .is_ok();
+
+            assert!(registered_role);
+
+            // Trigger event
+            Self::deposit_event(RawEvent::ChannelOwnershipTransferred(channel_id));
+        }
+
+        /// Channel owner updates some channel properties
+        pub fn update_channel_as_owner(
+            origin,
+            channel_id: ChannelId<T>,
+            new_channel_name: Option<Vec<u8>>,
+            new_description: Option<Vec<u8>>,
+            new_publishing_status: Option<ChannelPublishingStatus>) {
+
+            // Ensure channel owner has signed
+            Self::ensure_channel_owner_signed(origin, &channel_id)?;
+
+            // If set, ensure handle is acceptable length
+            if let Some(ref channel_name) = new_channel_name {
+                Self::ensure_channel_name_is_valid(channel_name)?;
+            }
+
+            // If set, ensure description is acceptable length
+            if let Some(ref description) = new_description {
+                Self::ensure_channel_description_is_valid(description)?;
+            }
+
+            //
+            // == MUTATION SAFE ==
+            //
+
+            Self::update_channel(
+                &channel_id,
+                &None,
```

Comment:
`new_channel_name` isn't used to update
```suggestion
                &new_channel_name,
```